nevergrad/functions/automl/test_automl.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import nevergrad as ng
+from nevergrad.benchmark.xpbase import Experiment
+
+from .core import AutoSKlearnBenchmark
+
+
```

Comment:
All these tests use some optimization, it would be nice to have at least one thaat just calls `func(whatever_parameters)` and checks that it works independently of anything else.